integrations/docker/images/base/chip-build/Dockerfile

```diff
@@ -152,6 +152,7 @@ RUN set -x \
     portpicker \
     pygit \
     PyGithub \
+    ruff \
```

Comment:
@agners since you change an image, you have to update the version file as well.

I would split this into two PS:
- first to change DockerFile and version
- 2nd that makes use of it

Reply:
Step 1: #32388

Reply:
Can we remove flake8? I understand it includes flake8 checks: site says `Drop-in parity with [Flake8](https://docs.astral.sh/ruff/faq/#how-does-ruff-compare-to-flake8), isort, and Black`

Reply:
Yes, right, that make sense!